fix: clarify help docs and command output for package-name flag (#1124)
salesforcecli/plugin-deploy-retrieve: messages/retrieve.start.md

# flags.package-name.summary

> Package names to retrieve. Note that this is for reference only and not meant to retrieve packaged metadata for development.

Reviewer: ```suggestion
Package names to retrieve. Use of this flag is for reference only; don't use it to retrieve packaged metadata for development.
```

# flags.package-name.description

> The metadata of the supplied package name(s) will be retrieved into a child directory of the project. The name of that child directory matches the name of the package. The retrieved metadata is meant for reference only and should not be added to a source control system for development and deployment. For package development the metadata should be retrieved using a manifest or by targeting a source controlled package directory within your project.

Reviewer: ```suggestion
The metadata of the supplied package name(s) will be retrieved into a child directory of the project. The name of that child directory matches the name of the package. The retrieved metadata is meant for your reference only, don't add it to a source control system for development and deployment. For package development, retrieve the metadata using a manifest (`--manifest` flag) or by targeting a source controlled package directory within your project (`--source-dir` flag).
```
